clients/da-vinci-client/src/test/java/com/linkedin/davinci/blobtransfer/TestNettyP2PBlobTransferManager.java

```diff
@@ -155,5 +186,13 @@ public void testLocalFileTransfer() throws IOException, ExecutionException, Inte
     Assert.assertTrue(Arrays.equals(Files.readAllBytes(file1), Files.readAllBytes(destFile1)));
     Assert.assertTrue(Arrays.equals(Files.readAllBytes(file2), Files.readAllBytes(destFile2)));
     Assert.assertTrue(Arrays.equals(Files.readAllBytes(file3), Files.readAllBytes(destFile3)));
+
+    // Verify the record is updated
+    Mockito.verify(storageMetadataService, Mockito.times(1))
+        .put(TEST_STORE + "_v" + TEST_VERSION, TEST_PARTITION, expectOffsetRecord);
+
+    // Verify the store version state is updated
+    Mockito.verify(storageMetadataService, Mockito.times(1))
```

Comment:
This verifies we updated the metadata on client side. Can we also add a few lines to ensure the metadata service is used to retrieve metadata?